content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Add new driver support matrix
+description: "Document to add new delfin driver support matrix"
+weight: 20
+disableToc: false
+tags: ["support matrix", "delfin", "drivers"]
+---
+
+Find below steps to add new delfin driver support matrix
+1) Clone the [documentation repository](https://github.com/sodafoundation/documentation.git)
```

Comment:
Should we put template with sample data first and then the steps to add new support matric page? @anvithks What do you think?

Reply:
Keep the sample data tables at the bottom of the page and give an in page link using the #id of the heading.